src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args){
+        Machine[] machines = {new Truck(),new Excavator(), new Bulldozer()};
 
+        for (Machine machine : machines){
```

Comment:
There should be a space after `for` and before the opening parenthesis `(` to maintain consistent code style.